server/services/screenshot_match.py

```
@@ -0,0 +1,268 @@
+
+    Reads the whole closet once and matches in Python — closets are small
+    (tens to low hundreds of items) and this avoids N per-garment queries.
+    """
+    try:
+        response = (
+            client.table("wardrobe_items")
+            .select(
+                "id, brand, item_name, color, image_url, "
+                "garment_type, garment_category, style"
+            )
+            .eq("is_available", True)
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)